architecture/additional_concepts/authentication.adoc

```diff
@@ -17,12 +17,46 @@ requesting user to determine if the request should be allowed.
 As an administrator, you can link:../../admin_guide/configuring_authentication.html[configure authentication] using a
 link:../../admin_guide/master_node_configuration.html[master configuration file].
 
+== Users and Groups
+
+A _user_ in OpenShift is an entity that can make requests to the OpenShift API.
+Typically, this represents the account of a developer or administrator that is
+interacting with OpenShift.
+
+A user can be assigned to one or more _groups_, each of which represent a
+certain set of users. Groups are useful for granting
+link:../../admin_guide/manage_authorization_policy.html[authorization policy] to
```

Comment:
Similar to comment above: "_the_ authorization policy" or "authorization _policies_".